src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -19,9 +19,13 @@ import {
     SIDEBAR_VIEW_SKILLS,
     SIDEBAR_VIEW_VERSIONS,
 } from '../../constants';
+import DetailsSidebar from './DetailsSidebar';
```

Comment:
You can simplify these a bit: `import DetailsSidebar, { type DetailsSidebarProps } from './DetailsSidebar';`